### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by ulises2607**
- Added press functionality to the hamburger icon displaying section titles.
- Added a close button in the navbar
- Modified nav section adding a "ul" tag for the list of the navigation
- Added main.js file
- Added Javascript functions to open and close the mobile menu
- Hamburger icon changed to bootstrap icon

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -47,16 +47,75 @@ header {
...
+    padding-bottom: 1rem;
+    margin-bottom: 2.5rem;
+    border-bottom: 1.5px grey solid;
+  }
+
+  .menu-desktop li a {
+    color: #fff5e1;
+    font-weight: 700;
+    font-size: 3rem;
+  }
```
Comment (AbelG101) [inline style.css:101]: - [ ] Ouch! this is close but there seems to be some discrepancy in your design of the options link relative to the Figma design. Take a closer look at the font and also remove the underline from the options link to stay consistent with the Figma design, I'd also advise you to center the horizontal line. As a front-end developer, you have to pay attention to small details like this as they provide a polished UI. ✨
- Figma design:
  image
- Your design:
  image

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @ulises2607 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Menu opens and closes as expected ✔
- Well-structured files ✔
- UI is looking great ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+const item = document.querySelectorAll('.element');
+
+openButton.addEventListener('click', () => {
+  nav.classList.add('visible');
+});
+
+closeButton.addEventListener('click', () => {
+  nav.classList.remove('visible');
+});
+
```
Comment (OyePriscilla) [inline main.js:13]: - [ ] Great developer! Kindly centralize the horizontal lines (`hr`) as pointed out by the previous reviewer, you are almost there. 👍 

## Your Design
Untitled

Figma Design
Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @ulises2607 👍 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Use JavaScript events ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @ulises2607 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 Linters are passing ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 README file is okay✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**